migrations/20191014111753-create-account.js

```diff
@@ -20,7 +20,7 @@ module.exports = {
       type: Sequelize.INTEGER,
       references: {
         model: 'Users',
-        key: 'membershipNo',
+        key: 'id',
```

Comment:
@albert85 Account should reference `membershipNo` on the Users and not the `id`

Reply:
@tijesunimi-peters can you please check the create the key title `userId`, I believed then it should be user (id) not user(membershipNo)